core/types/src/lib.rs

```diff
@@ -9,3 +9,5 @@ pub mod account;
 pub mod keys;
 
 pub mod markers;
+
```

Comment:
Could you open an issue to refactor this to follow [the style guide](https://github.com/mobilecoinfoundation/mobilecoin/blob/master/CONTRIBUTING.md#coding-style) before we release 5.0?